inc/templates/edit-form.tpl.php

```diff
@@ -52,7 +52,9 @@
 <script type="text/html" id="tmpl-shortcode-ui-field-textarea">
 	<div class="field-block">
 		<label for="{{ data.attr }}">{{ data.label }}</label>
-		<textarea name="{{ data.attr }}" id="{{ data.attr }}" placeholder="{{ data.placeholder }}">{{ data.value }}</textarea>
+		<textarea name="<#if ( data.attr ) { #>{{ data.attr }}<#}else{#>inner_content<#}#>" 
```

Comment:
Hm, does this mean I couldn't have `inner_content` as an attribute? Maybe we do need a separate template then.